### DISCUSSION TITLE
Indentation Error?

### DISCUSSION THREAD

**Discussion by mtess231989** [talkpython/python-for-absolute-beginners-course#48] [Q&A]
Hi everyone, I am fairly new to Python so I apologize if this is a dumb question. I do not understand why the program below will not run. Maybe indentation error?
import threading 

import queue 

import math 

 

q = queue.Queue() 

threads = [] 

 

def worker(): 

  while True: 

    item = q.get() 

    if item is None: 

    break 

    print(math.factorial(item)) 

    q.task_done() 

 

for x in range(5): 

  t                                           = threading.Thread(target=worker) 

  t.start() 

  threads.append(t) 

 

  zahlen = [134000, 14, 5, 300, 98, 88, 11, 23] 

  for item in zahlen: 

    q.put(item) 

 

  q.join() 

 

for i in range(5): 

   q.put(None)

Again, I am fairly new to Python and am just starting to learn. If you have any suggestion for me, please let me know.

Thank you,

**Comment by mikeckennedy** [comment]
Good luck with your homework. However, this forum is only for content related to the course.
